homework2/hw1.py

```diff
@@ -0,0 +1,115 @@
+# """
+# Given a file containing text. Complete using only default collections:
+#     1) Find 10 longest words consisting from largest amount of unique symbols
+#     2) Find rarest symbol for document
+#     3) Count every punctuation char
+#     4) Count every non ascii char
+#     5) Find most common non ascii char for document
+
+# """
+import re
+from typing import Dict, List
+
+
+def get_longest_diverse_words(file_path: str) -> List[str]:
+    text = ""
+    with open(file_path, "r") as fi:
+        text = fi.read()
+    if text == []:
+        return []
+    text = (
+        text.replace(".", "")
+        .replace(",", "")
+        .replace(":", "")
+        .replace("!", "")
+        .replace("?", "")
+        .replace(";", "")
+    )
+    wordlist = text.split()
```

Comment:
that works, but may be done simpler, try to use `string.punctuation` to cut off puncs